test/common.sh

```diff
@@ -44,7 +45,7 @@ if [ -n "$TRACE" ]; then
   set -x
 fi
 
-export KIND_K8S_VERSION="${KIND_K8S_VERSION:-"v1.22.1"}"
+export KIND_K8S_VERSION="${KIND_K8S_VERSION:-"v1.23.0"}"
```

Comment:
I think this also needs to change.

Reply:
I'm not sure if envtest has binaries for 1.23 released. Will check it out

Reply:
Seems like yes for ubuntu, but not for mac.

Reply:
Ah I see this: https://github.com/kubernetes-sigs/kubebuilder/issues/2474